codice/ddf
# DDF-2101 Created registry-federation-admin-impl to enable UI access to registry features

#### What does this PR do?
Adds the registry-federation-admin-impl module which provides an mbean for accessing/update registry information. This is an initial PR and can't be heroed or built until other peer PRs are merged.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@AzGoalie @spearskw 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@figliold 
@stustison
#### How should this be tested?
Outside of unit/itests 
1) Bring up an instance of DDF and start the DDF Registry app
2) Hit the endpoint https://localhost:8993/admin/jolokia/read/org.codice.ddf.registry:type=FederationAdminMBean/LocalNodes
3) Verify that you get a response back that has the following
 - one identity node in the 'nodes' array
 - six entries in the custom slots
 - two entries in the auto populate service bindings

#### Any background context you want to provide?
This module is coming over from the ddf-registry repo.
#### What are the relevant tickets?
DDF-2101
#### Screenshots (if appropriate)
#### Checklist:
